ci/build-win32.ps1

```diff
@@ -183,6 +187,42 @@ $projects = @(
     }
 )
 
+if ($arch -ne "arm64") {
+    $projects += @{
+        Path = "$subprojects/aom.wrap"
+        URL = "https://aomedia.googlesource.com/aom"
+        Revision = "main"
+        Method = "cmake"
+        Provides = @(
+            "aom = aom_dep"
+        )
+    }
+} else {
+    $projects += @{
+        Path = "$subprojects/aom-cmake.wrap"
+        URL = "https://aomedia.googlesource.com/aom"
+        Revision = "main"
+    }
+
+    # Manually wrap aom for AOM_TARGET_CPU option on ARM64
+    # aom auto detected the CPU architecture as `x86_64` on windows-11-arm action image. why?
```

Comment:
Because `windows-11-arm` image uses emulated x86 cmake. Could you instead install ARM cmake in build.yml? This should fix CPU detection.